sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/kafka/BindingServicePropertiesBeanPostProcessor.java

```diff
@@ -28,22 +33,36 @@ class BindingServicePropertiesBeanPostProcessor implements BeanPostProcessor {
     private static final String DEFAULT_KAFKA_BINDER_NAME = "kafka";
     private static final String KAFKA_BINDER_TYPE = "kafka";
 
+    private GenericApplicationContext applicationContext;
```

Comment:
Why using the GenericApplicationContext?

Reply:
Referred to the current code:
https://github.com/Azure/azure-sdk-for-java/blob/45f1e499518e530d913b831328bbd9174abcbb08/sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/jdbc/JdbcPropertiesBeanPostProcessor.java#L49

Reply:
Can we use ApplicationContext instead